README.md

```diff
@@ -12,17 +12,23 @@ Please see our [contributing guidelines](.github/CONTRIBUTING.md)
 
 ### [Installing](https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages#installing-a-package)
 
-1. Create `.npmrc` file in the root directory of your plugin project and fill with:
+* Install by running:
 
-    ```bash
-    @britecore:registry=https://npm.pkg.github.com/
+    ```
+    npm install @britecore/ui-plugins-client
     ```
 
-2. Install by running:
+* Use in the browser by adding:
 
+    ```html
+    <script src="https://unpkg.com/@britecore/ui-plugins-client@0.9.2/dist/britecore-ui-plugins.js"></script>
```

Comment:
why the full path, not https://unpkg.com/@britecore/ui-plugins-client.

Reply:
just to be explicit
we can certainly leave it off if you think that is better